SPECS-EXTENDED/rubygem-metaclass/rubygem-metaclass.spec

```diff
@@ -0,0 +1,152 @@
+# Generated from metaclass-0.0.1.gem by gem2rpm -*- rpm-spec -*-
```

Comment:
Same comment as in `rubygem-introspection`. Let's remove/update this comment.

Reply:
fixed